design-library/src/components/BccAlert/BccAlert.vue

```diff
@@ -1,15 +1,17 @@
 <script setup lang="ts">
 import {
-CloseIcon,
-ErrorFillIcon,
-InfoFillIcon,
-TaskAltIcon,
-WarningFillIcon
+  CloseIcon,
+  ErrorFillIcon,
+  InfoFillIcon,
+  TaskAltIcon,
+  WarningFillIcon,
+  NotificationsFillIcon,
 } from "@bcc-code/icons-vue";
+import type { Component } from "vue";
 
 type Props = {
   context?: keyof typeof contexts;
-  icon?: boolean;
+  icon?: boolean | Component;
```

Comment:
The type we use for `icon` in other components like the button is `string | Component | Function`, so it could be nice to add those other two as well to keep it similar.

Reply:
I'm actually not sure if string works on icon haven't tested it.

Reply:
Fair, and I think I ran into that when accidentally passing an icon as a string it would not throw an error because `string` is allowed (but would render `<searchicon>` as an HTML element), so maybe we should just remove it everywhere.